ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -661,9 +661,9 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
                                                              bool* result) {
     // TODO: replace current_backend ->
     // BackendManager::GetCurrentlySetBackendName()
-    auto config_map =
-        BackendManager::GetBackendAttributeValues(current_backend);
-    *result = (config_map.at("ngraph_backend") == "NNPI");
+    // FIXME(gopoka): GatherV2 is supported only when axis == 0.
+    //                If possible, check should be added here.
+    *result = true;
```

Comment:
please add `GatherV2` in `type_constraint_map`. 

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/ops.pbtxt#L16493

Tparams could be NGraphDTypes
Tindices and Taxis could be NGraphIndexDTypes

Reply:
`GatherV2` is already in `type_constraint_map`. This PR adds one special case when `axis == 0` (because this case can be handled by `GatherND` op in ngraph).

Reply:
Ah, right!